### PR TITLE
Hotfix/3 widgets fixed

### PR DESCRIPTION
**Pull request by szymon-owczarzak**

### CODE REVIEW

File: cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/WidgetIndex.kt [hunk 1]
```diff
@@ -28,6 +31,9 @@ class WidgetIndex {
             BambooDeploymentWidget::class.java.simpleName -> BambooDeploymentWidget(vertx, config)
             BambooPlanWidget::class.java.simpleName -> BambooPlanWidget(vertx, config)
             AemHealthcheckWidget::class.java.simpleName -> AemHealthcheckWidget(vertx, config)
+            TextWidget::class.java.simpleName -> TextWidget(vertx, config)
+            IframeEmbedWidget::class.java.simpleName -> IframeEmbedWidget(vertx, config)
+            WorldClockWidget::class.java.simpleName -> WorldClockWidget(vertx, config)
```
Comment (szymon-owczarzak) [inline cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/WidgetIndex.kt]: Or i can create only one new widget type like `CopyToContentWidget` that will have third argument with set of props to copy. WDYT ?

Comment (tMaxx) [reply cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/WidgetIndex.kt]: I'd leave as-is. By changing to what you describe, these props would have to be defined somewhere, and we would be essentially talking about placing the props as a class, or as a variable somewhere.

File: cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/BaseWidget.kt [hunk 2]
```diff
@@ -106,6 +106,14 @@ abstract class BaseWidget(
         return config
     }
 
+    protected fun updateStateByCopingPropsToContent(props: Set<String>) {
+        val content = JsonObject()
+        props.forEach {
```
Comment (tMaxx) [inline cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/BaseWidget.kt]: `props.map`?

Comment (szymon-owczarzak) [reply cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/BaseWidget.kt]: `forEach` will serve better in this case

**Pull request merged**